packages/babel-plugin-transform-regenerator/test/fixtures/issue-17517-destructuring-async-error/input.js

```diff
@@ -0,0 +1,13 @@
+async function testA() {
+  await Promise.resolve({
+    key: "value"
+  });
+  const {
+    resData
```

Comment:
Can you also add a test with nested destructuring?

```suggestion
    resData,
    other: { otherData: foo }
```